libs/libcds/src/hpx_tls_manager.cpp

```diff
@@ -0,0 +1,53 @@
+//  Copyright (c) 2020 Weile Wei
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0.
+//  (See accompanying file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+
+#include <hpx/libcds/hpx_tls_manager.hpp>
+#include <hpx/modules/threading.hpp>
+#include <hpx/threading_base/thread_data.hpp>
+
+#include <cds/gc/details/hp_common.h>
+
+#include <atomic>
+#include <cstddef>
+
+namespace cds { namespace gc { namespace hp { namespace details {
+
+    /*static*/ CDS_EXPORT_API thread_data* HPXDataHolder::getTLS()
```

Comment:
```suggestion
    CDS_EXPORT_API thread_data* HPXDataHolder::getTLS()
```
?

Reply:
We usually mark the declarations with the export macros, not the definition.

Reply:
Correct, it is done now.